Makefile

```diff
@@ -84,3 +84,7 @@ build-ui:
 
 rm-postgres:
 	rm -fr /tmp/quickwit/services/postgres
+
+start-prometheus:
```

Comment:
Do you want to make this part of our docker compose setup? You could then use `make docker-compose-up DOCKER_SERVICES=prometheus`